src/deploy/osp_deployer/settings/config.py

```diff
@@ -494,6 +496,8 @@ def __init__(self, settings_file):
             '/pilot/templates/dell-cinder-backends.yaml'
         self.dell_unity_cinder_yaml = self.foreman_configuration_scripts + \
             '/pilot/templates/dellemc-unity-cinder-backend.yaml'
+        self.ceph_osd_config_yaml = self.foreman_configuration_scripts + \
+            '/pilot/templates/ceph-osd-config.yaml'
```

Comment:
Did you intend to use CEPH_OSD_CONFIG_FILE (class variable) here?

Reply:
Oops!  Yes.  Fixed.